hazelcast/src/test/java/com/hazelcast/instance/HazelcastInstanceFactoryTest.java

```diff
@@ -43,62 +39,80 @@
 import static org.mockito.Mockito.doThrow;
 
 @RunWith(HazelcastParallelClassRunner.class)
-@Category({QuickTest.class, ParallelTest.class})
+@Category(QuickTest.class)
```

Comment:
I removed the `ParallelTest` category here, since I'm not sure that the created instances are all using the mocked network.